js/bittrex.js

```diff
@@ -400,38 +400,64 @@ module.exports = class bittrex extends Exchange {
     }
 
     parseTrade (trade, market = undefined) {
-        let timestamp = this.parse8601 (trade['TimeStamp'] + '+00:00');
-        let side = undefined;
-        if (trade['OrderType'] === 'BUY') {
-            side = 'buy';
-        } else if (trade['OrderType'] === 'SELL') {
-            side = 'sell';
-        }
-        let id = this.safeString2 (trade, 'Id', 'ID');
-        let symbol = undefined;
-        if (market !== undefined)
-            symbol = market['symbol'];
-        let cost = undefined;
-        let price = this.safeFloat (trade, 'Price');
-        let amount = this.safeFloat (trade, 'Quantity');
-        if (amount !== undefined) {
-            if (price !== undefined) {
-                cost = price * amount;
+        if ('Id' in trade) {
+            // then public market trade
+            let timestamp = this.parse8601 (trade['TimeStamp'] + '+00:00');
+            let side = undefined;
+            if (trade['OrderType'] === 'BUY') {
+                side = 'buy';
+            } else if (trade['OrderType'] === 'SELL') {
+                side = 'sell';
+            }
+            let id = this.safeString2 (trade, 'Id', 'ID');
+            let symbol = undefined;
+            if (market !== undefined)
+                symbol = market['symbol'];
+            let cost = undefined;
+            let price = this.safeFloat (trade, 'Price');
+            let amount = this.safeFloat (trade, 'Quantity');
+            if (amount !== undefined) {
+                if (price !== undefined) {
+                    cost = price * amount;
+                }
+            }
+            return {
+                'id': id,
+                'info': trade,
+                'timestamp': timestamp,
+                'datetime': this.iso8601 (timestamp),
+                'symbol': symbol,
+                'type': 'limit',
+                'side': side,
+                'price': price,
+                'amount': amount,
+                'cost': cost,
+                'fee': undefined,
+            };
+        } else {
+            // private market, ie, MY trades
+            let timestamp = this.parse8601 (trade['TimeStamp'] + '+00:00');
+            let marketInternal = this.marketsById[this.safeString (trade, 'Exchange')].symbol;
+            let amount = this.safeFloat (trade, 'Quantity');
+            let remaining = this.safeFloat (trade, 'QuantityRemaining');
+            let filled = undefined;
+            if (amount !== undefined && remaining !== undefined) {
+                filled = amount - remaining;
             }
+            return {
+                'id': this.safeString (trade, 'OrderUuid'),
```

Comment:
This should be saved into an `order` field, see the trade structure here:
https://github.com/ccxt/ccxt/wiki/Manual#trade-structure, the `id` should contain the trade id, not an order id...